server/src/controllers/game.controller.js

```diff
@@ -1,32 +1,105 @@
 // Package Modules
 const uuid = require("uuid");
+const { save, retrieve, deleteData } = require("../utils/cacheData");
 
 // Custom Modules
 const response = require("../utils/response");
 const CustomError = require("../utils/custom-error");
+const centrifugoController = require("../controllers/centrifugoController");
 
 class GameController {
-  async create(req, res) {
+  create(req, res) {
```

Comment:
you removed async here?
Any reason?

Reply:
There's nothing asynchronous in the function